content/php/concepts/math-functions/terms/decoct/decoct.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'decoct()'
+Description: 'Converts a decimal number to its octal representation.'
+Subjects:
+
+'Computer Science'
+'Web Development'
+Tags:
+'Arithmetic'
+'Functions'
+'Numbers'
+'Conversion'
+CatalogContent:
+'learn-php'
+'paths/web-development'
+
+---
+
+
+The **`decoct()`** function in PHP is used to convert a decimal number to its octal representation. It takes a decimal number as input and returns a string representing the octal equivalent.
+
+## Syntax
+```pseudo
+decoct(int $number): string
+```
+
+`$number`: The decimal number to be converted to octal.
+Returns a string representing the octal equivalent of the decimal number.
+
+## Example
+```php
+$decimalNumber = 123;
+$octalNumber = decoct($decimalNumber);
+echo $octalNumber;
+```
+
+The output for the above code will be:
+
+```shell
+173
+```
+
+In the above example, the decimal number `123` is converted to its octal equivalent, which is `173`.
```

Comment:
Move the description of the example to before the code block, after the example header.